lib/handlers.js

```diff
@@ -33,6 +34,10 @@ exports.schema = function(req, res, next) {
 };
 
 exports.get = function(req, res, next) {
+  if (req.params.id && !mongoose.Types.ObjectId.isValid(req.params.id)) {
+    exports.respond(res, 404, exports.objectNotFound());
+    next();
+  } else {
   req.quer.exec(function(err, list) {
```

Comment:
Sorry, I hate being nitpicky like this. Would you mind indenting the code inside of the `else` block?